app/scripts/lib/setupWeb3.js

```diff
@@ -1,26 +1,67 @@
+/*global Web3*/
 
 // TODO:deprecate:2020
+// Delete this file
 
-export default function setupDappAutoReload (web3, observable) {
+import 'web3/dist/web3.min.js'
+
+const shouldLogUsage = !([
+  'docs.metamask.io',
```

Comment:
Interesting - are these sites using `web3`?

Reply:
Ah right, the test dapp definitely is. For the others, it might be useful to know if they're using the injected `web3` instance though. That would be something we should fix.

Reply:
I swear to verify this manually 😄